test/conformance/api/v1/util.go

```diff
@@ -222,11 +222,11 @@ func validateControlPlane(t pkgTest.T, clients *test.Clients, names test.Resourc
 		if ready, err := v1test.IsRevisionReady(r); !ready {
 			return false, fmt.Errorf("revision %s did not become ready to serve traffic: %w", names.Revision, err)
 		}
-		if r.Status.ImageDigest == "" {
+		if r.Status.DeprecatedImageDigest == "" {
```

Comment:
(for conformance tests) Is adding validation of the new status field in scope for this PR or are you planning on adding it later?

oddly, enough we need to think about how it should be gated so that adding them doesn't cause conformance to fail when the multi-container feature is turned off

Reply:
Planning to conformance tests later

Reply:
thanks!